lib/components/user/user-account-screen.js

```diff
@@ -0,0 +1,422 @@
+import memoize from 'lodash.memoize'
+import PropTypes from 'prop-types'
+import React, { Component } from 'react'
+import { Navbar } from 'react-bootstrap'
+import { connect } from 'react-redux'
+
+import { addUser, fetchUser, updateUser } from '../../util/middleware'
+import { routeTo } from '../../actions/ui'
+import AppMenu from '../app/app-menu'
+
+import AccountCommands from './account-commands'
+import accountSetupFinishPane from './account-setup-finish-pane'
+import awaitingScreen from './awaiting-screen'
+import FavoriteLocationsPane from './favorite-locations-pane'
+import NotificationPrefsPane from './notification-prefs-pane'
+import phoneVerificationPane from './phone-verification-pane'
+import TermsOfUsePane from './terms-of-use-pane'
+import verifyEmailScreen from './verify-email-screen'
+
+function getStateForNewUser (auth0User) {
+  return {
+    auth0UserId: auth0User.sub,
+    email: auth0User.email,
+    hasConsentedToTerms: false, // User must agree to terms.
+    isEmailVerified: auth0User.email_verified,
+    notificationChannel: 'email',
+    phoneNumber: '',
+    recentLocations: [],
+    savedLocations: [],
+    storeTripHistory: false // User must opt in.
+  }
+}
+
+/**
+ * This screen handles creating/updating OTP user account settings.
+ */
+class UserAccountScreen extends Component {
```

Comment:
If possible, please split into two components as mentioned above. This file is very complex/long. Will review further after change is made.